packages/manager/apps/web/client/app/domain/zone/domain-zone-dns.html

```diff
@@ -28,6 +28,15 @@
                     data-translate="domain_tab_ZONE_no_zone_activate_btn"
                     data-ng-disabled="!ctrlDomainTabZoneDns.displayActivateZone"
                     data-ng-click="ctrlDomainTabZoneDns.activateZone()"
+                    data-ng-if="ctrlDomainTabZoneDns.region !== 'CA'"
```

Comment:
Could we use feature flipping here ?

Reply:
If i understand right the goal of FF is to enable/disable a feature/s according customer or region.
technically we can but in my case we wouldn't enable/disable just change workflow according the region.

Reply:
Maybe here we should see it as two features : 
- Ordering a zone attached to a domain
- Ordering a standalone DNS zone 
The first one is only available for EU whereas the second is available for both so maybe we could adapt the workflow depending on what feature is available :) (same as done here actually but with FF)

Reply:
Hello @JeremyDec and @marie-j, as i discussed with you, i will let him like this.
I created a Jira task to implement the feature flipping.
Thanks for your advices.